### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by vldtsw**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,14 @@
-# write your code here
+def file_creating():
+
+    file_name = input("Enter name of the file: ")
+
+    while True:
+        line = input("Enter new line of content: ")
+        if line == "stop":
+            break
+        with open(file_name + ".txt", "a") as f:
```
Comment (Abnormaltype) [inline app/main.py:9]: `f` is a bad name, better -> `file `

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

**Review by Oleksandr563487953 — APPROVED** [review]

File: app/main.py (hunk 1 above)
Comment (vldtsw) [reply app/main.py:9]: Fixed it

Comment (Abnormaltype) [reply app/main.py:9]: where commit?

Comment (vldtsw) [reply app/main.py]: Already here

**Review by Abnormaltype — APPROVED** [review]
Good job!

**Review by IvanRamyk — APPROVED** [review]

**Pull request closed without merging**